test/cloudant.test.js

```diff
@@ -45,18 +45,7 @@ describe('cloudant connector', function() {
       },
     });
 
-    db.automigrate(function cleanUpData(err) {
-      // automigrate only removes the design doc, but not instances' doc,
-      // so clean up data here just in case previous tests use same models.
-      if (err) return done(err);
-      Product.destroyAll(function removeModelInstances(err) {
-        if (err) return done(err);
-        CustomerSimple.destroyAll(function removeModelInstances(err) {
-          if (err) return done(err);
-          done();
-        });
-      });
-    });
+    db.automigrate(done);
```

Comment:
Good point, created a test case for it in https://github.com/strongloop/loopback-connector-cloudant/pull/109/files#diff-d4318f238579a6d222dac940aab8b69dR50